bin/test.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 
 down=false
-php="8.0"
```

Comment:
Sorry, this is my first PR to this project, but I couldn't get tests to run at all without this change, `bin/test.sh` would fail immediately with 
```
Your Composer dependencies require a PHP version ">= 8.1.0". You are running 8.0.24.
```

Reply:
This needs to be reverted. Laravel 9 still works on PHP 8.0 and needs to be tested as such with the lowest version.

Reply:
Thanks, have done.